bin/moonpick

```diff
@@ -20,45 +20,79 @@ Usage: moonpick [opts] <file>, [file2, ..]
 
 Where [opts] can be the following:
 
--v Shows moonpick version
--h Shows this help
+-v                Shows moonpick version
+-h                Shows this help
+--filename file   Filename used for lint_config with stdin
```

Comment:
Another more explicit way of doing things would be to allow for a `--config-file` argument specifying the configuration file to use. That would require more work on behalf of the caller though. Since the file in this scenario is known, I'm guessing that this is for easily integrating live linting of unsaved content for a file, e.g. in an editor?